skribenten-web/frontend/src/Brevredigering/LetterEditor/components/ContentGroup.tsx

```diff
@@ -176,7 +183,46 @@ export function EditableText({ literalIndex, content }: { literalIndex: LiteralI
     event.preventDefault();
     const nextFocus = allSpans[nextSpanIndex];
     nextFocus.focus();
-    selectService.focusAtOffset(nextFocus.childNodes[0], 0);
+    focusAtOffset(nextFocus.childNodes[0], 0);
+  };
+
+  const handleArrowUp = (event: React.KeyboardEvent<HTMLSpanElement>) => {
+    const element = contentEditableReference.current;
+    const caretCoordinates = getCaretRect();
+
+    if (element === null || caretCoordinates === undefined) {
+      return;
+    }
+
+    const shouldDoItOurselves = !areAnyContentEditableSiblingsPlacedHigher(element);
+
+    if (shouldDoItOurselves) {
+      const next = findOnLineAbove(element);
+
+      if (next) {
+        gotoCoordinates({ x: caretCoordinates.x, y: next.bottom - 10 });
```

Comment:
Hvor kommer `-10` fra her?

Reply:
Opplevde at hvis det var exact coordinate så "bommet" den iblandt. +10 og -10 var valgt litt random for å passe på at klikket skjer innafor elementet

Reply:
Kanskje bare ha det som en konstant med et navn som forklarer at dette er feilmargin e.l.?